backend/app/usecase/search/search.go

```diff
@@ -0,0 +1,19 @@
+package search
+
+import (
+	"short/app/entity"
+)
+
+// URLType can be public/private
+type URLType string
+
+// URLSearchResponse struct holds map of url corres. to their type
```

Comment:
please remove `struct` and use complete word instead of short cut to improve readability